src/region.js

```diff
@@ -223,7 +176,6 @@ var Region = MarionetteObject.extend({
     return Backbone.$(el, this.getValue(this.getOption('parentEl')));
   },
 
-  // Replace the region's DOM element with the view's DOM element.
   _replaceEl: function(view) {
     // empty el so we don't save any non-destroyed views
     this.$el.contents().detach();
```

Comment:
so, if we don't make `attachView` public, this isn't necessary.  Otherwise `attachView` becomes leaky if there's a view in there, right?  But honestly I think we needed `attachView` because show always re-rendered a view.  so I don't think there is a use-case anymore for `attachView`